src/components/books.js

```diff
@@ -22,42 +25,84 @@ const Books = () => {
       <h1>Loading...</h1>
     );
   }
+  const handlePercentage = () => {
+    let percentage = progress;
+    if (percentage < 100) {
+      setProgress(percentage += 1);
+    }
+  };
 
   const displayBooks = () => {
     if (Object.keys(theBooks).length > 0) {
-      return Object.keys(theBooks).map((id) => {
-        const mybook = theBooks[id][0];
-        if (!mybook) {
-          return null; // or handle the error in some other way
-        }
-        const { author, title, category } = mybook;
-        return (
-          <li key={id} id={id}>
-            <span className="author">{author}</span>
-            <span className="title">{title}</span>
-            <span className="category">{category}</span>
-            <button
-              type="submit"
-              className="removeBookButton"
-              onClick={() => {
-                handleRemoveBook(id);
-              }}
-            >
-              Remove
-            </button>
-          </li>
-        );
-      });
+      return (
+        <ul className="bookList">
+          {Object.keys(theBooks).map((id) => {
+            const [mybook] = theBooks[id];
+            const { author, title, category } = mybook;
+
+            return (
+              <li key={id} id={id} className="book">
+                <div className="detail-actions">
+                  <div className="bookDetails">
+                    <span className="author">{author}</span>
+                    <span className="title">{title}</span>
+                    <span className="category">{category}</span>
+                  </div>
+                </div>
+                <div className="actions">
+                  <button type="button" className="comment actionBtn">Comments</button>
+                  <button
+                    type="button"
+                    className="removeBook actionBtn"
+                    onClick={() => { handleRemoveBook(id); }}
+                  >
+                    Remove
+                  </button>
+                  <button type="button" className="actionBtn">Edit</button>
+                </div>
```

Comment:
- [ ] These buttons are expected to be placed at the bottom of the author's name. Kindly refer to the given design below.

**Your design**

<img width="1660" alt="Screenshot 2023-04-08 at 6 38 04 PM" src="https://user-images.githubusercontent.com/50941074/230737768-12f940e1-5623-4477-bcc3-c4224c81aafe.png">

**The Original Design**

<img width="1183" alt="Screenshot 2023-04-08 at 6 32 19 PM" src="https://user-images.githubusercontent.com/50941074/230737793-6b50d962-5849-4bd6-8652-8cada2f1df58.png">

Reply:
Done

Reply:
Fixed.